cmd/kops/validate_cluster.go

```diff
@@ -171,7 +171,8 @@ func RunValidateCluster(f *util.Factory, cmd *cobra.Command, args []string, out
 	} else {
 		// do we need to print which instance group is not ready?
 		// nodes are going to be a pain
-		fmt.Fprint(out, "\nValidation Failed\n")
+		// FIXME: format still not making sense
```

Comment:
LOL

>David Byrne is a Scottish-born American musician who was the founding member, principal songwriter, and lead singer and guitarist of the American new wave band Talking Heads, active between 1975 and 1991.

Him ^  ... This is notes to fix the damn thing.  The output sucks.  WIP